drivers/src/data_vault.rs

```diff
@@ -518,7 +518,7 @@ impl DataVault {
     ///
     pub fn read_warm_reset_entry48(&self, entry: WarmResetEntry48) -> Array4x12 {
         let dv = self.dv.regs();
-        Array4x12::read_from_reg(dv.nonsticky_data_vault_entry().at(entry.into()))
+        Array4x12::read_from_reg(dv.sticky_data_vault_entry().at(entry.into()))
```

Comment:
This is wrong. I think nonsticky_data_vault_entry was renamed data_vault_entry().

Reply:
Same elsewhere